lib/plugins/aws/deploy-function.js

```diff
@@ -227,6 +227,12 @@ class AwsDeployFunction {
       delete params.KMSKeyArn;
     }
 
+    if (functionObj.snapStart) {
```

Comment:
Can you confirm what happens if we first deploy with `snapStart: true` and then redeploy with this setting removed (or set to `false`).

Will the SnapStart configuration be removed from lambda?

Reply:
**SnapStart : true**

<img width="533" alt="Screenshot 2022-12-08 at 21 44 16" src="https://user-images.githubusercontent.com/2007116/206639311-7507a2dc-d744-438d-b0e6-dd28985e36fc.png">
<img width="1655" alt="Screenshot 2022-12-08 at 21 44 23" src="https://user-images.githubusercontent.com/2007116/206639308-9b675802-c68b-4020-8a85-c336791f4a59.png">

**SnapStart : false**

<img width="699" alt="Screenshot 2022-12-08 at 21 51 21" src="https://user-images.githubusercontent.com/2007116/206639425-c7c0b5be-7d33-40c6-84f9-0525e9605675.png">
<img width="1570" alt="Screenshot 2022-12-08 at 21 51 24" src="https://user-images.githubusercontent.com/2007116/206639400-d5a6a2e9-75d7-4249-adc9-73c74c7101f0.png">

Reply:
@debae `snapStart: false` should have same behavior as not setting `snapStart` at all.

Can you confirm what happens if we first deploy with `snapStart: true` and then redeploy with this setting removed?

Reply:
@debae Can you confirm what happens if we first deploy with snapStart: true and then redeploy with this setting removed?

It's really important to confirm on that, to be sure our implementation is solid. Thank you!